config/config.go

```diff
@@ -369,9 +370,12 @@ func Load(paths []string, conf *Config) error {
 		log.Info("Loading default configuration...")
 		return Load([]string{path}, conf)
 	}
-	if err := json.NewDecoder(r).Decode(&conf); err != nil {
+
+	bytes, _ := ioutil.ReadAll(r)
+	if err := yaml.Unmarshal(bytes, &conf); err != nil {
```

Comment:
As mentioned in slack, I would rather not rely on on 3-rd party library for json parsing, and will run `yaml.Unmarshal` only if string looks like YAML, since it is easy to detect by checking first byte.